examples/01-Detailed-Examples/05-mesh-exploration.py

```diff
@@ -0,0 +1,209 @@
+"""
+.. _ref_mesh_exploration_example:
+
+Explore the mesh
+================
+In this script a static simulation is used as an example to show how to
+query mesh information such as connectivity, element IDs, element types and so on.
+"""
+from __future__ import annotations
+
+from ansys.dpf import post
+from ansys.dpf.post import examples
+from ansys.dpf.post.common import elemental_properties
+
+###############################################################################
+# Perform required imports
+# ------------------------
+# Perform required imports. # This example uses a supplied file that you can
+# get by importing the DPF ``examples`` package.
+
+
+###############################################################################
+# Get ``Simulation`` object
+# -------------------------
+# Get the ``Simulation`` object that allows access to the result. The ``Simulation``
+# object must be instantiated with the path for the result file. For example,
+# ``"C:/Users/user/my_result.rst"`` on Windows or ``"/home/user/my_result.rst"``
+# on Linux.
+
+example_path = examples.download_all_kinds_of_complexity()
+simulation = post.StaticMechanicalSimulation(example_path)
+
+# print the simulation to get an overview of what's available
+print(simulation)
+stress_df = simulation.stress(components=["XX"])
+print(stress_df)
+
+###############################################################################
+# Get the mesh
+# ------------
+
+# Retrieve the actual mesh
+mesh = simulation.mesh
+
+###############################################################################
+# Query basic information about the mesh
+# --------------------------------------
+
+# Node IDs
+n_ids = mesh.node_ids
+
+# Element IDs
+e_ids = mesh.element_ids
+
+# Available named selection names
+named_selections = mesh.available_named_selections
```

Comment:
Can we remove available_named_selections if we have named_selections with all the info please?